### DISCUSSION TITLE
multi team tools

### DISCUSSION THREAD

**Discussion by Willyfrog** [mattermost/community#45] [Practical Ideas]
have tools that span its actions across the different teams
- channel switcher: so if I'm using the channel switcher I also get channels from the other teams (with some indication of that) so 
- search posts: same as above, it will also require some sort of quick button to restrict the search to the current team if one doesn't want to spread the search.
- others (use the comments below to add to the list)

**Comment by chetanyakan** [comment]
- unread channels
- multi-team threads
- Creating cross-team pinned channel categories in LHS

**Comment by agarciamontoro** [comment]
I'd *love* to see this. I think @crspeller had a proof of concept for the channel switcher (or maybe for the search?).

**Reply by crspeller** [reply]
Yup: https://github.com/mattermost/mattermost-webapp/pull/7217

**Comment by esethna** [comment]
Ya we really need to work on our cross-team UX. It has been an afterthought for years, but it's certainly a "productivity win" we need to prioritize. For any devs interested, we'll be doing a productivity wins "hackathon" soon so let's get a team on this ;)
